tests/test_circuit/test_reg_enable_call.py

```diff
@@ -1,11 +1,13 @@
 import magma as m
+from magma.generator import ParamDict
 
 
 def test_reg_enable_call():
     class test_reg_enable_call(m.Circuit):
         io = m.IO(I=m.In(m.Bits[5]), O=m.Out(m.Bits[5]), nen=m.In(m.Bit))
         io += m.ClockIO()
-        reg = m.Register(m.Bits[5], has_enable=True, enable_name="en")()
+        reg = m.Register(m.Bits[5], has_enable=True,
+                         name_map=ParamDict(CE="en"))()
```

Comment:
Does the `name_map` need to be a `ParamDict` or will any mapping work?

Reply:
Needs to be hashable (ParamDict is just a dict that defines a __hash__ method)